app/services/nomis/elite2/movement_api.rb

```diff
@@ -26,6 +26,15 @@ def self.movements_for(offender_no)
           api_deserialiser.deserialise(Nomis::Movement, movement)
         }
       end
+
+      def self.latest_movement_for(offender_no)
```

Comment:
Seems a shame to have the same code as the other function, might be worth replacing with something like ...

```suggestion
    movements_for(offender_no)&.first
```

Reply:
The issue is that the API doesn't return the movements in a date ordered list, so you can't guarantee that the first/last thing in the list is going to actually be the latest movement